tools/ExtensionLibrary/DevHome.ExtensionLibrary/Assets/extensionResult.json

```diff
@@ -0,0 +1,35 @@
+{
+  "ProductIds": ["9NZCC27PR6N6", "9N8MHTPHNGVV"],
+  "Products":
+  [
+    {
+      "LocalizedProperties":
+	  [
+        {
+          "PublisherName": "Microsoft Corporation",
+          "ProductTitle": "Dev Home GitHub Extension (Preview)"
+        }
+      ],
+      "ProductId": "9NZCC27PR6N6",
+      "Properties":
+	  {
+        "PackageFamilyName": "Microsoft.Windows.DevHomeGitHubExtension_8wekyb3d8bbwe"
+      }
+    },
+    {
+      "LocalizedProperties": 
+	  [
+        {
+          "PublisherName": "Microsoft Corporation",
+          "ProductTitle": "Dev Home (Preview)"

```

Comment:
Why are we adding Dev Home (Preview) as an extension?

Reply:
Never mind, I see we're ignoring it later on.  This is just to mimic what the store will return to us in the future.  Perfect!